src/components/external/DisputeManagement/components/DefendDisputeFlow/DefendDisputeFlow.scss

```
@@ -0,0 +1,74 @@
+@use '../../../../../style';
+
+.adyen-pe-defend-dispute {
+    &__container {
+        display: flex;
+        flex-direction: column;
+        gap: style.token(spacer-070);
+        padding-top: style.token(spacer-070);
+        width: 100%;
+    }
+
+    &-document-requirements {
+        font-family: system-ui;
+
+        &--list {
+            margin: 0;
+            padding-left: style.token(spacer-070);
+        }
+    }
+
+    &__title {
+        color: #000;
+        font-weight: 700;
+    }
+
+    &__error-icon {
+        color: style.token(color-background-critical-strong);
+        height: style.token(spacer-100);
+        width: style.token(spacer-100);
+
+        svg {
+            height: inherit;
+            width: inherit;
+        }
```

I think what you need for the `svg` element here is some style declaration(s) to get rid of the vertical alignment weirdness with `svg` elements.

To get the desired `svg` icon size, setting the correct font size on the `svg` parent element should suffice, since the `svg` element already has a `height` property value of `1em` (coming from the `<Icon />` component).

```suggestion
        color: style.token(color-background-critical-strong);
        font-size: style.token(spacer-100);

        svg {
            display: block;
        }
```